### ISSUE TITLE
Bug Report: Several cli args are not used in `m4b-util bind`

### ISSUE THREAD

**Issue by AndyRightNow** [Tsubashi/m4b-util#1]
First off, thanks for your great work on this tool!

When using this command, the cli args `--use-filename` and `--decode-durations` don't seem to have any effect.

```bash
$ m4b-util bind INPUT_DIR --use-filename
```

After digging into the installed PIP package, I found out that the cli args aren't passed into the process function. 

https://github.com/Tsubashi/m4b-util/blob/dff6dd8075f2dc4233b49d43a4f3fcc3f47a5d52/src/m4b_util/subcommands/bind.py#L62

Currently I just manually modify the installed PIP package code to get it to work. It works finne for my case but I figured it might help to report the bug.

I haven't used other commands yet but there might also be similar issues.

**Comment by Tsubashi** [comment]
Good catch! Apparently I didn't commit all my changes when I implemented that feature. It is committed now in 1a4e614ff6779445da8de8eb40ffaadd20bbfcb9 and I'll make a new release as soon as I get the test runner sorted out. 

Thanks for reporting this, and please don't hesitate to reach out if you find more bugs or have any feature that would make the tool even better!

**Comment by Tsubashi** [comment]
The fix is out in the latest release [2025.1.6](https://github.com/Tsubashi/m4b-util/releases/tag/2025.1.6). Again, thanks for reporting the issue!
